release.yaml

```diff
@@ -151,4 +151,3 @@ api:
     obr: true
     bom: true
     mvp: true
-    isolated: true
```

Comment:
We'll need to keep dev.galasa.framework in isolated, this might've been an accidental removal.

Also, looks like you're missing a version bump for the `dev.galasa.framework.api.openapi` bundle in the release.yaml since it's now at 0.34.0